### PR TITLE
Added custom build variants

### PR DESCRIPTION
**Pull request by dborowiecki**
In this PR I've added build variants for demo instances in gradle file. 
Build variants were added for: 
 - MV
 - BEPHA
 - CHF

PR also includes new strings for these variants.

### CODE REVIEW

**Review by delcroip — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -50,30 +50,65 @@ android {
         buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
         buildConfigField "boolean", "SHOW_CONTROL_NUMBER_MENU", 'false'
         buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
-        resValue "string", "app_name", "openIMIS Policies"
```
Comment (dragos-dobre) [inline app/build.gradle:53]: Why did you removed the app name from configuration? We are using this to make sure every build variant has another name and, if installed on same smartphone, can differentiate between apps. It also help to know which version you are running. Please add this and adapt the names for the new BV. The app_name is removed from the translation files. 
As it gives an error when having this in string.xml and build.graddle, is there a solution to replace the app name in string.xml when this is present in build.graddle?
https://stackoverflow.com/questions/24785270/how-to-change-app-name-per-gradle-build-type/36455983

File: app/build.gradle [hunk 2]
```diff
@@ -50,30 +50,65 @@ android {
...
     }
     productFlavors {
         demoProd {
             applicationId "org.openimis.imispolicies.demoProd"
             buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
             buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
-            resValue "string", "app_name", "Policies Demo"
```
Comment (dragos-dobre) [inline app/build.gradle:60]: Restore this.

File: app/build.gradle [hunk 3]
```diff
@@ -50,30 +50,65 @@ android {
...
             dimension = 'std'
         }
+
         demoRelease {
             applicationId "org.openimis.imispolicies.demoRelease"
             buildConfigField "String", "API_BASE_URL", '"http://release.openimis.org/rest/"'
+        }
+
+        chfDev {
```
Comment (dragos-dobre) [inline app/build.gradle:67]: Add app_name for all.

File: app/build.gradle [hunk 4]
```diff
@@ -50,30 +50,65 @@ android {
...
+            buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
             dimension = 'std'
             versionCode = gitVersionCodeTime
         }
     }
     sourceSets {
         demoProd.java.srcDir 'src/demo/java'
         demoProd.res.srcDir 'src/demo/res'
+
         demoRelease.java.srcDir 'src/demo/java'
```
Comment (dragos-dobre) [inline app/build.gradle:99]: You can combine the source sets if they have same folders.
```
[demoProd, demoRelease].each {
            it.java.srcDir 'src/demo/java'
            it.res.srcDir 'src/demo/res'
        }
```

File: app/src/demo/res/values-fr/strings.xml [hunk 5]
```diff
@@ -387,4 +387,5 @@
   <string name="date_to">Date à</string>
   <string name="send_sms">Envoyer un SMS</string>
   <string name="requested_sms">SMS demandé</string>
+  <string name="app_name">openIMIS POLICE</string>
```
Comment (dragos-dobre) [inline app/src/demo/res/values-fr/strings.xml:390]: Remove this if in build.graddle, unless solution found to replace this value.

File: app/build.gradle [hunk 6]
```diff
@@ -50,30 +50,65 @@ android {
...
+
         demoRelease {
             applicationId "org.openimis.imispolicies.demoRelease"
             buildConfigField "String", "API_BASE_URL", '"http://release.openimis.org/rest/"'
+        }
+
+        chfDev {
+            applicationId "org.openimis.imispolicies.chf"
+            buildConfigField "String", "API_BASE_URL", '"http://chf-dev.swisstph-mis.ch/rest/"'
+            buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
```
Comment (dragos-dobre) [inline app/build.gradle:70]: Remove this here.

File: app/build.gradle [hunk 7]
```diff
@@ -50,30 +50,65 @@ android {
...
+            buildConfigField "String", "API_BASE_URL", '"http://imis-mv.swisstph-mis.ch/rest/"'
+            buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
             buildConfigField "boolean", "SHOW_CONTROL_NUMBER_MENU", 'true'
-            resValue "string", "app_name", "Policies Release"
+            dimension = 'std'
+        }
+        bephaDev {
+            applicationId "org.openimis.imispolicies.bepha"
+            buildConfigField "String", "API_BASE_URL", '"http://149.210.235.40/devupgrade/rest/"'
+            buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
```
Comment (dragos-dobre) [inline app/build.gradle:84]: Remove this here.

File: app/build.gradle [hunk 8]
```diff
@@ -50,30 +50,65 @@ android {
...
+        mvDev {
+            applicationId "org.openimis.imispolicies.mv"
+            buildConfigField "String", "API_BASE_URL", '"http://imis-mv.swisstph-mis.ch/rest/"'
+            buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
             buildConfigField "boolean", "SHOW_CONTROL_NUMBER_MENU", 'true'
-            resValue "string", "app_name", "Policies Release"
+            dimension = 'std'
+        }
+        bephaDev {
+            applicationId "org.openimis.imispolicies.bepha"
```
Comment (dragos-dobre) [inline app/build.gradle:82]: name to bephaDev

File: app/build.gradle [hunk 9]
```diff
@@ -50,30 +50,65 @@ android {
...
+            buildConfigField "String", "API_BASE_URL", '"http://149.210.235.40/devupgrade/rest/"'
```
Comment (dragos-dobre) [inline app/build.gradle:83]: create bephaProd BV and updte the API_BASE_URL.

**Pull request merged**